### PR TITLE
MSPSDS-871: Non-office homepage

### PR DESCRIPTION
**Pull request by MatthewBurstein**
A couple of questions about this PR:
- Because the redirect for non-office users happens in the investigations controller, the root route is still /cases. Is there a better way to do this?
- Similarly the location and name of non_office_index.slim.html probably aren't great. Is there convention for where this should live and what it should be called?

### CODE REVIEW

File: mspsds-web/app/controllers/investigations_controller.rb [hunk 1]
```diff
@@ -8,6 +8,11 @@ class InvestigationsController < ApplicationController
   # GET /cases.json
   # GET /cases.xlsx
   def index
+    # TODO delete unless block when user role exists for The Office
+    if !current_user.is_office?
+      render "non_office_index"
```
Comment (kicferk1) [inline mspsds-web/app/controllers/investigations_controller.rb]: Does this render prevent the following lines from executing? In particular, do we query elasticsearch for investigations?

File: mspsds-web/app/models/user.rb [hunk 2]
```diff
@@ -62,6 +62,10 @@ def has_role?(role)
     KeycloakClient.instance.has_role? role
   end
 
+  def is_office?
+    KeycloakClient.instance.has_role? :the_office
```
Comment (kicferk1) [inline mspsds-web/app/models/user.rb]: Do any of test users actually have this role? Do we need to add it in keycloak console?

Comment (broder) [reply mspsds-web/app/models/user.rb]: We'll need to add it to the `initial-setup.json` as well. Also, is something like `office_user`, `opss_user` or `opss_employee` a better name for the role?

Comment (broder) [reply mspsds-web/app/models/user.rb]: @kicferk1, should the role in https://github.com/UKGovernmentBEIS/beis-mspsds/pull/455/files#diff-4d9a28c3cbbb7360d78650adc7a5cc78R102 be the same as whatever we call the `office` role here?

Comment (kicferk1) [reply mspsds-web/app/models/user.rb]: I think so.

File: mspsds-web/app/controllers/investigations_controller.rb (hunk 1 above)
Comment (MatthewBurstein) [reply mspsds-web/app/controllers/investigations_controller.rb]: Good spot, I've added a return statement here.

**Comment by kicferk1** [comment]
Sorry for all the conflicts, they are probably my fault :P 
Happy to pair up on fixing them, it ought to be pretty straightforward.

File: mspsds-web/app/controllers/investigations_controller.rb [hunk 3]
```diff
@@ -8,6 +8,11 @@ class InvestigationsController < ApplicationController
   # GET /cases.json
   # GET /cases.xlsx
   def index
+    # TODO delete unless block when user role exists for The Office
+    if !current_user.is_office?
```
[the remark this answers is not in the document]
Comment (MatthewBurstein) [reply mspsds-web/app/controllers/investigations_controller.rb]: Yes makes perfect sense :)

File: mspsds-web/app/controllers/homepage_controller.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+class HomepageController < ApplicationController
+  def show
+    if current_user.is_opss?
+      redirect_to investigations_path
+    elsif !current_user.is_opss?
```
Comment (broder) [inline mspsds-web/app/controllers/homepage_controller.rb]: Is there a particular reason that this isn't just `else`?

Comment (MatthewBurstein) [reply mspsds-web/app/controllers/homepage_controller.rb]: Nope 👍

File: mspsds-web/app/assets/stylesheets/navbar.scss [hunk 5]
```diff
@@ -0,0 +1,65 @@
+// Taken from https://github.com/alphagov/govuk-design-system/blob/master/src/stylesheets/components/_navigation.scss
```
Comment (broder) [inline mspsds-web/app/assets/stylesheets/navbar.scss]: Do we need this and the one in `shared-web`?

Comment (MatthewBurstein) [reply mspsds-web/app/assets/stylesheets/navbar.scss]: 👍 I've added my changes to the file in shared-web

**Pull request merged**